uniprot-search/src/main/java/uk/ac/ebi/uniprot/search/document/suggest/SuggestDocument.java

```diff
@@ -0,0 +1,40 @@
+package uk.ac.ebi.uniprot.search.document.suggest;
+
+
+import lombok.*;
+import org.apache.solr.client.solrj.beans.Field;
+import uk.ac.ebi.uniprot.search.document.Document;
+
+import java.util.List;
+
+
+@EqualsAndHashCode
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Do we use AllArgsConstructor?

Reply:
ah, accidentally left this here! not needed so removed now, thanks!